CONTRIBUTING.md

```diff
@@ -0,0 +1,167 @@
+The first part, `issue-7/` is fixed and unchanging, the tailing short description could be different (`no-trailing-symbols` for example).
+
+### Issue Branch merging
+
+When an issue branch is merged with the develop branch, it is deleted, resulting in the complete loss of the development history of an issue.
+The goal is to reduce the amount of data to sort through to get to meaningful information.
+It's important to comment on all commits to assist reviewers.
+Many changes are small/short updates (or at least should be) and aren't important after reviewing and merging.
+If there's something important to be considered in the future, it should be documented elsewhere.
+
+This policy may be amended in the future.
+
+## Documentation
+
+You are responsible for providing new documentation for your changes.
+Depending upon what the changes are, it may be necessary to create brand new documentation files.
+
+## Testing
+
+You are responsible for providing testing criteria and environments for indicating success.
+This may include providing source files, expected output, any acceptable variations on the output, and needed documentation.
+
+## Tags
+
+We are following [Semantic Versioning](https://semver.org/) for version numbers.
+Every released version has a matching tag associated with it.
+
+## Workflow
+
+1. Make sure you have a [GitHub account](https://github.com/signup/free)
+2. If you have a new feature or improvement, please discuss your ideas with the community on the [Slack](https://terra-ref.slack.com) channel.
+3. Submit a ticket in [GitHub](https://github.com/terraref/drone-pipeline/issues) for your issue, if one doesn't exist already
+  * If it's a bug, use the `Issue Report` issue template and clearly describe the issue as outlined
+  * If it's a new feature or an improvement, use the `Feature Request` issue template and clearly describe all requirements and how your proposed solution would fulfill them. For complex
+  new features it might help to create a [wiki page](https://github.com/terraref/drone-pipeline/wiki)
+4. Create a fork on [GitHub](https://github.com/terraref/drone-pipeline)
+5. Check out the `develop` branch
+6. Make a feature/issue/task branch using the naming convention described above
+7. Make your cool new feature or bug fix on your branch and update the documentation
+8. From your branch, make a pull request against `develop`, being sure to point out where to find documentation updates (clearly labeled URIs are fine), and providing testing materials
+9. Work with repo maintainers to get your change reviewed on [GitHub](https://github.com/terraref/drone-pipeline)
+10. After approval, your branch will be merged into the `develop` branch
+11. Your branch will be deleted soon after being merged. Post-merge issues or changes are addressed with new tickets.
+
+## Code Reviews
+
+All code reviews will be part of the pull request and take place on GitHub.
+
+## Continuous Integration
+
+(TBD)
+
+## Code of Conduct
```

Comment:
I think we could have a global code of conduct for the terraref organization or this can be put on our main website and referenced from the contributing guidelines. Both contributing and code of conduct should be referenced from the overall README

Reply:
Broken out into CODEOFCONDUCT.md